coreax/kernel.py

```diff
@@ -24,7 +24,7 @@
 points, we can reconstruct the underlying data generating distribution through smoothing
 of the data with kernels.
 
-Some kernels are parameterizable and may represent other well known kernels when given
```

Comment:
The old spelling is correct British English as defined by the OED.

Reply:
Weird, fixed